extensions/identity-hub-verifier/src/main/java/org/eclipse/dataspaceconnector/identityhub/verifier/DidJwtCredentialsVerifier.java

```diff
@@ -60,6 +61,7 @@ public boolean isSignedByIssuer(SignedJWT jwt) {
         return signatureResult.succeeded();
     }
 
+    @Override
```

Comment:
nit: You can merge main to avoid spurious diff as this have been already fixed in the last identityhub commit in main.